app/src/controllers/testItem/utils.js

```diff
@@ -121,3 +121,13 @@ export const groupItemsByParent = (items) =>
       [item.parent]: [...group, item],
     };
   }, {});
+
+export const isItemOwner = (userId, item, launch) => {
+  if (item.owner) {
```

Comment:
Should we convince that `item` exists?

Reply:
It is required to use this utility with the ʻitem` property.
It is called ʻisItemOwner` to check the user's ownership of the corresponding item.
Therefore, it is the developer's responsibility to provide the required arguments to use this feature.

Reply:
And it is just copied from https://github.com/reportportal/service-ui/pull/2406/files#diff-dd4a7cd981a131a9b1d016eefd80f869R238